nvflare/app_common/executors/splitnn_learner_executor.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pt.learners.cifar10_learner_splitnn import SplitNNConstants
+
+from nvflare.apis.event_type import EventType
+from nvflare.apis.executor import Executor
+from nvflare.apis.fl_constant import ReturnCode
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable, make_reply
+from nvflare.apis.signal import Signal
+from nvflare.app_common.abstract.learner_spec import Learner
+
+
+class SplitNNLearnerExecutor(Executor):
```

Comment:
As discussed with @chesterxgchen and @ZiyueXu77, we might want to have just one LearnerExecutor for all situations if possible.

Rather than having multiple: RandomForestLearnerExecutor, StatisticsLearnerExecutor, XGBoostLearnerExecutor.

But this depends on whether they do indeed have common interfaces.

Reply:
But nothing needs to be done in this PR for this class.

This is just a note.

Reply:
It's worth discussing. Created an issue to track this point #1353.